server/plugins/FeatureDetection.cpp

```diff
@@ -61,6 +61,14 @@ struct RunningSum : public Unit {
 	float* msquares;
 };
 
+// like RunningSum, but with variable size summing window. - mtmccrea
+struct RunningSum2 : public Unit {
```

Comment:
i recommend adding new UGens in separate C++ source files for navigability. i know that almost no UGens that we have right now conform to this, but i'll refactor 'em all eventually...